tests/src/test/scala/whisk/core/controller/test/PackagesApiTests.scala

```diff
@@ -370,6 +370,20 @@ class PackagesApiTests extends ControllerTestCommon with WhiskPackagesApi {
     }
   }
 
+  it should "reject create package when package name is 'default'" in {
```

Comment:
Is there a test for update of an existing package named default?

Reply:
not currently; i'll add one